test/login.test.ts

```diff
@@ -12,25 +14,29 @@ describe('User login endpoint', async function () {
   before(async () => {
     connection = getConnection();
     userRepository = connection.getRepository(UserEntity);
-    await connection.synchronize();
   });
 
   beforeEach(async () => {
     await userRepository.clear();
   });
 
   it('succesfully logs in', async () => {
+    const user = userRepository.create(new UserFixture());
+    user.password = bcrypt.hashSync(user.password, 10);
+
+    await userRepository.save(user);
     const response = await request.post('graphql').send({
-      query: `mutation { login(loginInput: { email: "email" password: "password" }){
-      user { id, name, email, birthDate } token }}`,
+      query: `mutation { login(loginInput: { email: "${user.email}", password: "${new UserFixture().password}" }){
+      user { id, name email birthDate } token }}`,
     });
+    const birthDate = new Date(user.birthDate);
 
     expect(response.body.data.login).to.be.deep.eq({
       user: {
-        id: 1,
-        name: 'User Name',
-        email: 'User e-mail',
-        birthDate: new Date('04-25-1990').toISOString(),
+        id: user.id,
+        name: user.name,
+        email: user.email,
+        birthDate: `${birthDate.getFullYear()}-0${birthDate.getMonth() + 1}-${birthDate.getDate()}T00:00:00.000Z`,
```

Comment:
Se o mês for 10 ou 11 não dá ruim essa checagem?

Reply:
No PR #28 eu mudei o type dessa coluna e não acontece mais esse problema de ter que ficar dando parse na data.
Tive problemas pq o typeorm retorna Date como string e sempre tem que ficar inicializando um objeto Date se eu precisar usar algum método de Date. Enfim, já está alterado isso na pr 28.